### PR TITLE
Adding in disable button state when submitting activity message form

### PR DESCRIPTION
**Pull request by Guntrisoft**
* Using the built in ujs disable button functionality

### CODE REVIEW

**Comment by Guntrisoft** [comment]
@benlovell

File: app/views/activities/_activity_feed.html.erb [hunk 1]
```diff
@@ -8,7 +8,7 @@
           <div class="input-group">
             <%= f.text_field :message, placeholder: 'Add an action to keep others in the loop', class: 'form-control js-message t-message' %>
             <span class="input-group-btn">
-              <%= f.button class: 'btn btn-primary btn-block t-submit-message' do %>
+              <%= f.button class: 'btn btn-primary btn-block t-submit-message', :data => { :disable_with => 'Adding message...' } do %>
```
Comment (benlovell) [inline app/views/activities/_activity_feed.html.erb]: Can you change the hash rockets to 1.9 style keys please?

**Comment by Guntrisoft** [comment]
@benlovell re-jigged my under performing PR

**Comment by benlovell** [comment]
Das ist gut.

**Pull request merged**